docker-compose.test.yml

```
@@ -40,3 +53,8 @@
     ports:
       - '8080:8080'
     restart: unless-stopped
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:8080" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)